cmd/cored/main.go

```diff
@@ -176,12 +177,7 @@ func runServer() {
 	if conf != nil {
```

Comment:
Hm, now I wonder whether this `if/else` shouldn't be encapsulated inside a single `launchCore` function, and maybe it should be called `newHandler`? In any case, still your call, still LGTM.